README.md

```diff
@@ -1,18 +1,1481 @@
+
+```javascript
+authClient.session.exists()
+.then(function(exists) {
+  if (exists) {
+    // logged in
+  } else {
+    // not logged in
+  }
+});
+```
+
+## session.get()
+
+Gets the active [session](http://developer.okta.com/docs/api/resources/sessions.html#example).
+
+```javascript
+authClient.session.get()
+.then(function(session) {
+  // logged in
+})
+.catch(function(err) {
+  // not logged in
+});
+```
+
+## session.refresh()
+
+Refresh the current session by extending its lifetime. This can be used as a keep-alive operation.
+
+```javascript
+authClient.session.get()
+.then(function(session) {
+  // existing session is now refreshed
+})
+.catch(function(err) {
+  // there was a problem refreshing (the user may not have an existing session)
+});
+```
+
+## session.close()
+
+Signs the user out of their current Okta session.
+
+```javascript
+signIn.session.close()
+.then(function() {
+  // the user has been logged out
+})
+.catch(function(err) {
+  // the user has not been logged out, perform some error handling here.
+});
+```
+
+## token.getWithoutPrompt(oauthOptions)
+
+When you've obtained a sessionToken from the authorization flows, or a session already exists, you can obtain a token or tokens without prompting the user to log in.
+
+  - `oauthOptions` - See [OIDC Configuration](http://localhost:6419/#openid-connect-options)
+
+```javascript
+authClient.token.getWithoutPrompt({
+  responseType: 'id_token', // or array of types
+  sessionToken: 'testSessionToken' // optional if the user has an existing Okta session
+})
+.then(function(tokenOrTokens) {
+  // manage token or tokens
+})
+.catch(function(err) {
+  // handle OAuthError
+});
+```
+
+## token.getWithPopup(oauthOptions)
+
+Create token with a popup.
+
+  - `oauthOptions` - See [OIDC Configuration](http://localhost:6419/#openid-connect-options)
+
+```javascript
+authClient.token.getWithPopup(oauthOptions)
+.then(function(tokenOrTokens) {
+  // manage token or tokens
+})
+.catch(function(err) {
+  // handle OAuthError
+});
+```
+
+## token.getWithRedirect(options)
+
+Create token using a redirect.
+
+  - `oauthOptions` - See [OIDC Configuration](http://localhost:6419/#openid-connect-options)
+
+```javascript
+authClient.token.getWithRedirect(oauthOptions);
+```
+
+## token.parseFromUrl(options)
+
+Parses the access or ID Tokens from the url after a successful authentication redirect.
+
+```javascript
+authClient.token.parseFromUrl()
+.then(function(tokenOrTokens) {
+  // manage token or tokens
+})
+.catch(function(err) {
+  // handle OAuthError
+});
+```
+
+## token.decode(idTokenString)
+
+Decode a raw ID Token
+
+  - `idTokenString` - an id_token jwt
+
+```javascript
+authClient.token.decode('YOUR_ID_TOKEN_JWT');
+```
+
+## token.refresh(tokenToRefresh)
+
+Returns a new token if the Okta session is still valid.
+
+  - `tokenToRefresh` - an access token or ID token previously provided by Okta. note: this is not the raw jwt
```

Comment:
nit: jwt -> JWT

Later on we should make this less confusing, and not call what we're returning and setting a "token" - it would just be the options that we're setting, where not all of them are/should be required (like claims).